src/pymor/basic.py

```diff
@@ -39,18 +39,15 @@
 from pymor.domaindescriptions.basic import RectDomain, CylindricalDomain, TorusDomain, LineDomain, CircleDomain
 from pymor.domaindescriptions.polygonal import DiscDomain, CircularSectorDomain, PolygonalDomain
 
-from pymor.domaindiscretizers.default import discretize_domain_default
-
-from pymor.discretizers.cg import discretize_stationary_cg, discretize_instationary_cg
-from pymor.discretizers.fv import discretize_stationary_fv, discretize_instationary_fv
+from pymor.discretizers.builtin import (discretize_stationary_cg, discretize_instationary_cg,
+                                        discretize_stationary_fv, discretize_instationary_fv,
+                                        OnedGrid, TriaGrid, RectGrid, load_gmsh)
+from pymor.discretizers.builtin.domaindiscretizers.default import discretize_domain_default
+from pymor.discretizers.builtin.grids.boundaryinfos import EmptyBoundaryInfo, GenericBoundaryInfo, AllDirichletBoundaryInfo
```

Comment:
we could even drop this and add some `pymor.discretizers.basic` which would result in

```python
from pymor.basic import *
from pymor.discretizers.basic import *
```

Its only one additional line in interactive sessions but would further document the splitting.

Reply:
Or just

```python
from pymor.basic import *
from pymor.discretizers.builtin import *
```